libs/sdk-ui-dashboard/src/presentation/filterBar/filterBar/DefaultFilterBarContainer.tsx

```diff
@@ -39,6 +51,7 @@ const DefaultFilterBarContainerCore: React.FC<{ children?: React.ReactNode }> =
             >
                 <AllFiltersContainer setCalculatedRows={setCalculatedRows}>{children}</AllFiltersContainer>
                 <FiltersRows rows={rows} />
+                <CrossFilteringConfiguration />
```

Comment:
Can we make this more general? To me it looks like a general filter configuration rather than just cross filtering configuration.